### PR TITLE
Parse maintainers info

### PR DESCRIPTION
**Pull request by garrrikkotua**
# Changes proposed ✍️

### What
copilot:summary
​
copilot:poem

### How
copilot:walkthrough

## Checklist ✅
- [ ] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screehshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

File: crowdgit/cm_maintainers_data/evaluate_maintainers.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+from urllib.parse import urlparse
```
Comment (joanreyero) [inline crowdgit/cm_maintainers_data/evaluate_maintainers.py:1]: We don't need this file anymore, no?

File: crowdgit/cm_maintainers_data/normalise_roles.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from cm_database import query, execute
```
Comment (joanreyero) [inline crowdgit/cm_maintainers_data/normalise_roles.py:1]: Same with this.

File: crowdgit/maintainers.py [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+        except Exception as error:
+            logger.error(f"Error processing repo {url}: {error}")
+            write_failed_repo(url, owner, repo_name, str(error))
+
+
+async def run():
+    async with asyncio.TaskGroup() as tg:
+        tg.create_task(parse_not_parsed())
+        # tg.create_task(parse_already_parsed())
+        # tg.create_task(reidentify_repos_with_no_maintainer_file())
```
Comment (joanreyero) [inline crowdgit/maintainers.py:288]: Also several things here that we do not need.

**Review by joanreyero — COMMENTED** [review]
The logic looks good. However, I still see several pieces of code from my analysis that should not be needed here. Could we remove this?

File: crowdgit/maintainers.py (hunk 3 above)
Comment (garrrikkotua) [reply crowdgit/maintainers.py:288]: Which ones? Cost related stuff?

**Pull request merged**